src/components/sections/robots/robot.tsx

```diff
@@ -0,0 +1,182 @@
+import "@/styles/globals.css";
+import Image from "next/image";
+
+export default function SponsorImages2023() {
+  const logoFolder = "https://cdn.team2658.org/web-public/robots"; // last character CANNOT be a '/'

```

Comment:
download the images where you can't see the robots on to your machine, crop them or modify them in a way so that you can, and then send them to me in a google drive folder and i will upload them to the CDN.